src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -964,119 +974,115 @@ bool processAreaScopeV2(const Scope* scoP, orion::BSONObjBuilder* queryP, bool a
   orion::BSONObjBuilder bobArea;
   if (scoP->georel.type == "near")
   {
```

Comment:
For this part, better to use "hide whitespace" option in GitHub

NTC (just a hint)